packages/requirements.txt

```diff
@@ -1,2 +1,2 @@
-django==1.2 
+django==1.4.11 
```

Comment:
### :warning: Aqua detected vulnerability in your code  
**Vulnerability ID:** CVE-2015-5963   
**Check Name:** python-django: Denial-of-service possibility in logout() view by filling session store   
**Severity:** MEDIUM   
**Fixed Version:** 1.8.4, 1.7.10, 1.4.22   
**Description:** contrib.sessions.middleware.SessionMiddleware in Django 1.8.x before 1.8.4, 1.7.x before 1.7.10, 1.4.x before 1.4.22, and possibly other versions allows remote attackers to cause a denial of service (session store consumption or session record removal) via a large number of requests to contrib.auth.views.logout, which triggers the creation of an empty session record.  
[This comment was created by Aqua Pipeline]  
  
Read more at https://avd.aquasec.com/nvd/cve-2015-5963